src/oop/oop1.py

```diff
@@ -17,3 +17,32 @@
 #     pass
 #
 # Put a comment noting which class is the base class
```

Comment:
these classes are fine, they just break the tests since the tests expect no parameters being passed in to each class.  

Also be careful with your spelling and make sure you're ending `def` lines with a `:`.  You'll get better with this as you use Python more, but for now check VSCode for a python syntax highlighter/linter to catch these mistakes.